Police-Data-Accessibility-Project/design-system
# feat(components): dropdown (#80)

Adds dropdown
- #64 

## To test
1. Pull this branch, run the demo app locally with `npm run dev`
2. Check the behavior, styling, and accessibility of the dropdown component.
